src/core/components/SearchModal.tsx

```diff
@@ -66,9 +68,9 @@ const SearchModal = ({
   return (
     <div className={styles.container}>
       <button
```

Comment:
Is this button just in the PR so the modal can be opened while refactor is in progress? Not I'd expect this to be a layer outside the modal I think so the parent opens the modal.

Reply:
This component is the button and the modal it launches.